cleanup.py

```diff
@@ -1,14 +1,14 @@
 import os
 import base64
-import requests
 import json
 from datetime import date, timedelta
+import requests
 
 url = "https://midaxo.jfrog.io/midaxo"
 username = os.environ['ARTIFACTORY_USERNAME']
 password = os.environ['ARTIFACTORY_PASSWORD']
-credentials = base64.b64encode(username + ":" + password)
-headers = {'Authorization': 'Basic ' + credentials}
+credentials = base64.b64encode(username.encode() + b":" + password.encode())
```

Comment:
what is the `b` for?

Reply:
It signifies to python3 that it is a bytes object as opposed to a string, and base64.b64encode only supports encoding bytes-objects. 

In python2 this wasn't needed because strings are stored as bytes objects by default, however in python3 the default is unicode string, if prefixed with `b''` it will work in both versions of python 😄